hypertrace-core-graphql-grpc-utils/src/main/java/org/hypertrace/core/graphql/utils/grpc/DefaultGrpcChannelRegistry.java

```diff
@@ -1,44 +1,23 @@
 package org.hypertrace.core.graphql.utils.grpc;
 
 import io.grpc.ManagedChannel;
-import io.grpc.ManagedChannelBuilder;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
 import javax.inject.Inject;
 import javax.inject.Singleton;
 import org.hypertrace.core.graphql.spi.lifecycle.GraphQlServiceLifecycle;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 @Singleton
 class DefaultGrpcChannelRegistry implements GrpcChannelRegistry {
-  private static final Logger LOG = LoggerFactory.getLogger(DefaultGrpcChannelRegistry.class);
-  private final Map<String, ManagedChannel> channelMap = new ConcurrentHashMap<>();
-  private volatile boolean isShutdown = false;
+
+  private final org.hypertrace.core.grpcutils.client.GrpcChannelRegistry delegate =
```

Comment:
nit: style

Reply:
missed this in the review

Reply:
what specifically about the style (this was formatted by spotless)? the FQN?

Reply:
yes. never mind. i see it now.